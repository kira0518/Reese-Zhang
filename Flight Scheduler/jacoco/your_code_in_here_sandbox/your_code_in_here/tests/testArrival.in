location import locations.csv
flight import flights6.csv
Arrivals
Arrivals adfasdas
Arrivals Sydney
exit

