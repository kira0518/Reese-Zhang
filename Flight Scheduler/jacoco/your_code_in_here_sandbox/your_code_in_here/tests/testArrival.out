User: Imported 51 locations.

User: Imported 70 flights.
112 lines were invalid.

User: This location does not exist in the system.

User: This location does not exist in the system.

User: Sydney
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
  47 Mon 19:35   Arrival from Sydney
  39 Mon 20:27   Arrival from Sydney
  40 Mon 22:57   Arrival from Sydney
  33 Tue 21:47   Arrival from Sydney
   0 Wed 21:25   Arrival from Sydney
   5 Thu 02:44   Arrival from Sydney
  51 Fri 17:29   Arrival from Sydney
  49 Sat 08:31   Arrival from Sydney
  50 Sat 10:31   Arrival from Sydney
  48 Sat 13:31   Arrival from Sydney

User: Application closed.
